frontend/src/scenes/sessions/filters/EditFiltersPanel.tsx

```diff
@@ -1,56 +1,89 @@
 import { CloseButton } from 'lib/components/CloseButton'
 import { EventTypePropertyFilter, PersonPropertyFilter, RecordingPropertyFilter } from '~/types'
 import { sessionsFiltersLogic } from 'scenes/sessions/filters/sessionsFiltersLogic'
 import { EventPropertyFilter } from 'scenes/sessions/filters/EventPropertyFilter'
 import { PersonFilter } from 'scenes/sessions/filters/UserFilter'
 import { DurationFilter } from 'scenes/sessions/filters/DurationFilter'
+import { SessionsFilterBox } from 'scenes/sessions/filters/SessionsFilterBox'
+import { AddFilterButton } from 'scenes/sessions/filters/AddFilterButton'
 
 interface Props {
     onSubmit: () => void
 }
 
-const SECTIONS: Record<string, { label: string; description: string }> = {
+const SECTIONS: Record<string, { label: string; description: JSX.Element }> = {
     recording: {
         label: 'Recording filters',
-        description: 'Find sessions with recordings matching the following',
+        description: (
+            <>
+                Find sessions <b>with recordings</b> matching the following
+            </>
+        ),
     },
     person: {
         label: 'User property filters',
-        description: 'Find sessions where user properties match the following',
+        description: (
+            <>
+                Find sessions where user properties match <b>all</b> of the following
+            </>
+        ),
     },
     action_type: {
         label: 'Action filters',
-        description: 'Find sessions where user has done a given action',
+        description: (
+            <>
+                Find sessions where a given <b>action</b> has been triggered
+            </>
+        ),
     },
     event_type: {
         label: 'Event filters',
-        description: 'Find sessions where user has done a given event',
+        description: (
+            <>
+                Find sessions where a given <b>event</b> has been triggered
+            </>
+        ),
     },
     cohort: {
         label: 'Cohort filters',
-        description: 'Find sessions by users in the following cohorts',
+        description: (
+            <>
+                Find sessions by <b>users in</b> the following cohorts
+            </>
+        ),
     },
 }
 
-export function EditFiltersPanel({ onSubmit }: Props): JSX.Element {
-    const { displayedFilters } = useValues(sessionsFiltersLogic)
-    const { openFilterSelect, removeFilter } = useActions(sessionsFiltersLogic)
+export function EditFiltersPanel({ onSubmit }: Props): JSX.Element | null {
+    const { activeFilter, displayedFilterCount, displayedFilters } = useValues(sessionsFiltersLogic)
+    const { openFilterSelect, openEditFilter, removeFilter } = useActions(sessionsFiltersLogic)
+
+    if (displayedFilterCount === 0) {
+        return null
+    }
+
+    const andTag = (visible: boolean): JSX.Element => (
+        <Tag className="and-tag" style={{ visibility: visible ? 'initial' : 'hidden' }}>
```

Comment:
There's already a global class for this, `.stateful-badge .and`, we just need to merge #2959 first.